draft-ietf-ppm-dap.md

```diff
@@ -152,6 +152,16 @@ input is ever seen in the clear by any aggregator.
 
 12:
 
+- Bump draft-irtf-cfrg-vdaf-08 to 12 {{!VDAF}}. (\*)
+
+- Add support for "asynchronous" aggregation, based on the Leader polling the
+  Helper for completion. (\*)
+
+- Update collection semantics to match the new aggregation semantics introduced
+  in support of asynchronous aggregation. (\*)
+
+- Clarify the requirements around report replay protection.
```

Comment:
This refers to b99bcfd1c08786550147934640044af492cb9579, correct? Let's add a bit more context here. This bullet should answer the question: "do I need to change my implementation in some way?"

Reply:
I've updated the text a bit to add detail -- let me know what you think. I think implementations might need to change due to this -- e.g. Janus doesn't (I think), but it's plausible that there were other implementation strategies that no longer fit what is specified for replay protection with this change.